runtime/connection_cache.go

```diff
@@ -121,8 +121,12 @@ func (r *Runtime) openAndMigrate(ctx context.Context, cfg cachedConnectionConfig
 			cfg.config = maps.Clone(cfg.config)
 			cfg.config["managed"] = true
 
-			if inst.AdminConnector == "" {
-				// Provisioning has been requested, but the instance does not have an admin connector.
+			// For DuckDB with managed: true, we don't need admin provisioning.
+			// Skip admin provisioning to avoid errors in cloud environments where the admin service doesn't have a DuckDB provisioner.
+			skipAdminProvisioning := cfg.driver == "duckdb" && cfg.config["managed"] == true
```

Comment:
Doesn't make sense to check `cfg.config["managed"] == true` here, it's always going to be `true` per line 122